config/config.go

```diff
@@ -81,7 +81,7 @@ func getKubeAPIServerURL() string {
 	if p == "" {
 		// we still want the simulator to behave as before,
 		// use a local test port.
```

Comment:
Remove this line.

Reply:
fixed: [f4deb1d](https://github.com/kubernetes-sigs/kube-scheduler-simulator/pull/115/commits/f4deb1d59c6dbe294e4d9f4ddd27f61ca990dab0)